frontend/tests/ui/test_course.py

```diff
@@ -158,14 +148,14 @@ def show_run_details_course(driver: conftest.Driver, course_alias: str,
     enter_course_assignments_page(driver, course_alias)
     with driver.page_transition():
         driver.wait.until(EC.element_to_be_clickable(
-            (By.XPATH, '//a[@href = "/course/%s/assignment/%s/admin/"]'
+            (By.XPATH, '//a[@href = "/course/%s/assignment/%s/"]'
              % (course_alias, assignment_alias)))).click()
 
     util.show_run_details(driver,
-                          table_classname='local',
+                          table_classname='runs',
                           dropdown_classname='open',
                           code='#include <iostream>',
-                          has_been_migrated=False)
+                          has_been_migrated=True)
 
     driver.browser.find_element_by_css_selector('#overlay').click()
```

Comment:
Este elemento, aunque aún existe, parece que ya no se requiere. Ahora se está utilizando otro, así que ese selector se puede sustituir por:
```suggestion
    driver.browser.find_element_by_css_selector('div[data-overlay]').click()
```